rovercode/input_utils.py

```diff
@@ -0,0 +1,60 @@
+"""Helper functions for managing inputs."""
+
+import logging
+
+import constants
+
+from binary_sensor import BinarySensor
+from drivers.grovepi_ultrasonic_ranger_binary \
+    import GrovePiUltrasonicRangerBinary
+
+logging.basicConfig()
+LOGGER = logging.getLogger(__name__)
+LOGGER.setLevel(logging.getLevelName('INFO'))
+
+
+def init_inputs(rover_params):
+    """Initialize input GPIO."""
+    binary_sensors = []
+
+    left_port, right_port = \
+        rover_params.get(constants.LEFT_ULTRASONIC_PORT),\
+        rover_params.get(constants.RIGHT_ULTRASONIC_PORT)
+    if any((left_port is None, right_port is None)):
+        raise ValueError('Required sensor port configuration'
+                         'not present in Rover config')
+    try:
+        left_port = int(left_port)
+        right_port = int(right_port)
+    except ValueError as error:
+        LOGGER.error("Unusable values %s and %s for ultrasonic ports",
+                     left_port, right_port)
+        raise error  # TODO: Set RGB LED to red
```

Comment:
If you just want to re-raise the current exception you can just put `raise`. Then, you don't need the variable

Reply:
Nice! Fixing.